applications/luci-app-mwan3/luasrc/model/cbi/mwan/interfaceconfig.lua

```diff
@@ -107,9 +107,9 @@ family = mwan_interface:option(ListValue, "family", translate("Internet Protocol
 	family:value("ipv4", translate("IPv4"))
 	family:value("ipv6", translate("IPv6"))
 
-track_ip = mwan_interface:option(DynamicList, "track_ip", translate("Tracking IP"),
-	translate("This IP address will be pinged to dermine if the link is up or down. Leave blank to assume interface is always online"))
-	track_ip.datatype = "ipaddr"
+track_ip = mwan_interface:option(DynamicList, "track_ip", translate("Tracking host"),
+	translate("This host will be pinged to dermine if the link is up or down. Leave blank to assume interface is always online"))
```

Comment:
old typo:
dermine --> determine

You are breaking translations in any case, so you might add a hint that both hostname and ip address are valid.